src/api/spec/support/coverage.rb

```diff
@@ -11,5 +11,9 @@
   add_filter '/lib/templates/'
   add_filter '/lib/memory_debugger.rb'
   add_filter '/lib/memory_dumper.rb'
+
+  # filter spec directory not in sync with test/test_helper
```

Comment:
I don't understand the comment. What is not in sync?

Reply:
if you open the file, you will see two lines above, the following comment:

"NOTE: Keep filters in sync with test/test_helper.rb"